backend/api/routes/files.py

```diff
@@ -147,6 +151,52 @@ def validate_project_quota(file_size: int, project: Project):
         )
 
 
+def s3_file_key(project_id: UUID, path: str) -> str:
+    """Generate s3 file key."""
+    return f"{project_id}-{path}"
```

Comment:
I can finish it in this PR. I'v think about it and I think the most straightforward way is using the salted hash. We can set a pre-defined salt and add this salt to `{project_id}-{path}` and do sha256 once again. The key of a file will not be guessable since the salt is hidden. 
Please let me know if other projects are in a similar situation.